tests/drivers/dac/dac_api/testcase.yaml

```diff
@@ -5,5 +5,6 @@ tests:
     platform_allow:
       frdm_k22f frdm_k64f nucleo_l073rz nucleo_l152re twr_ke18f nucleo_f767zi
       nucleo_f429zi bl652_dvk bl653_dvk bl654_dvk bl5340_dvk_cpuapp stm32f3_disco
-      stm32l562e_dk nucleo_l552ze_q b_u585i_iot02a
+      stm32l562e_dk nucleo_l552ze_q b_u585i_iot02a nucleo_g474re nucleo_g071rb
+      nucleo_wl55jc nucleo_h743zi nucleo_l476rg nucleo_f091rc nucleo_f207zg
```

Comment:
It would be nice to add new boards in alphabetical order, same as the defines.

Reply:
changing to `nucleo_f207zg nucleo_g474re nucleo_h743zi nucleo_l476rg nucleo_wl55jc`

Reply:
Still neither alphabetical order nor same order as in the `#define`s in `tests/drivers/dac/dac_api/src/test_dac.c`...